README.md

```diff
@@ -44,7 +44,7 @@ Used in the `author` field in `package.json`
 ---
 
 ### ☁️ Include API-helper?
-If you select this, `source/js/api-helper.js` will be included. This is a handy abstraction of `fetch` that supports automating analytics (optional), showing status messages (optional) and working with mock API responses.
+If you select this, `source/js/api-helper.js` will be included. This is a handy abstraction of `fetch` that supports automating analytics (optional), showing status messages (optional) and using json files to mock API requests.
 
```

Comment:
`responses` is correct here. We're not mocking the requests, as they are very real.

Reply:
Yeah you're right